synadm/api.py

```diff
@@ -557,7 +557,10 @@ def user_modify(self, user_id, password, display_name, threepid,
         if deactivation == "activate":
             data.update({"deactivated": False})
         if user_type:
-            data.update({"user_type": user_type})
+            if user_type == 'null':
```

Comment:
could be instead:
```python
data.update({"user_type": None if user_type == 'null' else user_type})
```

Reply:
yeah that's nicer. I like oneliners like that but will check if the linelength is sufficient......

Reply:
Oh, well that fits easily. Changed it. Thanks for the suggestion. Much better well....much less ugly now ;-))